client/sysinfo/src/sysinfo.rs

```diff
@@ -121,7 +184,7 @@ pub const DEFAULT_CPU_EXECUTION_LIMIT: ExecutionLimit =
 	ExecutionLimit::Both { max_iterations: 4 * 1024, max_duration: Duration::from_millis(100) };
 
 // This benchmarks the CPU speed as measured by calculating BLAKE2b-256 hashes, in MB/s.
```

Comment:
```suggestion
// This benchmarks the CPU speed as measured by calculating BLAKE2b-256 hashes, in bytes per second.
```

Same for all the other mentions where it previously said that it was in MB/s.